R/utilities-configuration-plan.R

```diff
@@ -214,3 +272,28 @@ tlfScale <- function(configurationScale) {
   # tolower is used to ensure that there is no issue with caps from field values
   ConfigurationScales[[tolower(configurationScale)]]
 }
+
+#' @title get
+#' @description Translate value of `Scaling` property from configuration plan into tlf `scale`
+#' @param pathArray Configuration plan path as an array for observed data
+#' @param configurationPlan configuration plan object
+#' @param simulation simulation object
+#' @return molecular weight in base unit or NA if not found
+getMolWeightForObservedData <- function(pathArray, configurationPlan, simulation) {

```

Comment:
please refactor and pass observedDataId, compoundName etc. as arguments instead of pathArray, e.g.

`getMolWeightForObservedData <- function(observedDataId, compoundName, configurationPlan, simulation) `